src/vi/ungtycomics/src/eu/kanade/tachiyomi/extension/vi/ungtycomics/UngTyComics.kt

```diff
@@ -184,7 +184,7 @@ class UngTyComics : ParsedHttpSource(), ConfigurableSource {
 }
 
 private const val PREF_BASE_URL = "baseUrl"
-private val MIRRORS = arrayOf("https://ungtycomicsvip.com", "https://topdammy.com")
+private val MIRRORS = arrayOf("https://www.ungtycomics.io", "https://ungtycomicsvip.com", "https://topdammy.com")
```

Comment:
Why keep mirrors if they return 404?

Reply:
> Why keep mirrors if they return 404?

Problem with SSL, so i added fix base in https://github.com/keiyoushi/extensions-source/pull/4844